Tests/Shared.Specs/TypeAssertionSpecs.cs

```diff
@@ -484,6 +484,65 @@ public void When_an_unrelated_type_instance_it_fails_with_a_useful_message()
                 .WithMessage($"*{typeof(DummyImplementingClass)} to be assignable to {typeof(DateTime)}*failure message*");
         }
 
+        [Fact]
+        public void When_constructed_of_open_generic_it_succeeds()
```

Comment:
🙃  We normally use the `When_...._it_should_....` format for all our tests. 
🤔 I don't get the _when constructed_. We're not constructing anything here, right?